pkg/virt-operator/install-strategy/create.go

```diff
@@ -3358,3 +3264,56 @@ func createOrUpdateKubeVirtCAConfigMap(
 	}
 	return nil, nil
 }
+
+func removeKvServiceAccountsFromDefaultSCC(targetNamespace string,
+	clientset kubecli.KubevirtClient,
+	stores util.Stores) error {
+
+	modified := false
+
+	SCCObj, exists, err := stores.SCCCache.GetByKey("privileged")
+	if !exists {
+		return nil
+	} else if err != nil {
+		return err
+	}
+
+	SCC, ok := SCCObj.(*secv1.SecurityContextConstraints)
+	if !ok {
+		return fmt.Errorf("couldn't cast object to SecurityContextConstraints: %+v", SCCObj)
+	}
+
+	oldUsers := SCC.Users
+	SCCCopy := SCC.DeepCopy()
+	users := SCCCopy.Users
+
+	kvServiceAccounts := rbac.GetKubevirtComponentsServiceAccounts(targetNamespace)
+	for _, acc := range oldUsers {
+		if contains(kvServiceAccounts, acc) {
```

Comment:
If you will create a map from the users, you will reduce the runtime of this loop from O(n^2) to O(n)

Reply:
yeah, so if you had a map of the kubevirt users, it would just be a direct lookup.

```
if ok := kvSericeAccounts[acc]; ok {


}
```

Reply:
@danielBelenky @davidvossel Thank you. Done.